### PR TITLE
Add Edit on Public and Private address label and Limit min and max sc…

### PR DESCRIPTION
**Pull request by vikas-cis**
…ale size of electron window

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -44,6 +44,8 @@ function initMainWindow(trayImage) {
   mainWindow = new BrowserWindow({
     width: 1280,
     height: 720,
+    minWidth: 961,
```
Comment (rynomster) [inline main.js]: Please remove these.. Some people want full-screen on UHD.. Others want mobile type view

File: src/app/wallet/receive/modals/add-address-label/add-address-label.component.html [hunk 2]
```diff
@@ -7,7 +7,7 @@ <h2 md-dialog-title>Label for new {{ type }} address</h2>
   <md-dialog-actions>
     <button md-button md-dialog-close>Cancel</button>
     <button md-button [disabled]="!addLableForm.valid" md-raised-button color="primary">
-      Add Label
```
Comment (rynomster) [inline src/app/wallet/receive/modals/add-address-label/add-address-label.component.html]: Make use of ngIf or something, no need for dynamic text string..

File: src/app/wallet/receive/receive.component.ts [hunk 3]
```diff
@@ -353,9 +353,12 @@ export class ReceiveComponent implements OnInit {
       error => this.log.er('error'));
     }
   }
-  openNewAddress(): void {
+  openNewAddress(callType: string, text: string, add: string): void {
```
Comment (rynomster) [inline src/app/wallet/receive/receive.component.ts]: Make these optional

File: main.js (hunk 1 above)
Comment (vikas-cis) [reply main.js]: aah ok

**Pull request merged**